deployment/keystone/contract_set.go

```diff
@@ -92,3 +92,19 @@ func DeployForwarder(lggr logger.Logger, chain deployment.Chain, ab deployment.A
 	lggr.Infof("Deployed %s chain selector %d addr %s", forwarderResp.Tv.String(), chain.Selector, forwarderResp.Address.String())
 	return nil
 }
+
+// DeployFeedsConsumer deploys the KeystoneFeedsConsumer contract to the chain
+// and saves the address in the address book. This mutates the address book.
+func DeployFeedsConsumer(lggr logger.Logger, chain deployment.Chain, ab deployment.AddressBook) error {
+	consumerDeploy := KeystoneFeedsConsumerDeployer{lggr: lggr}
+	consumerResp, err := consumerDeploy.deploy(DeployRequest{Chain: chain})
+	if err != nil {
+		return fmt.Errorf("failed to deploy FeedsConsumer: %w", err)
+	}
+	err = ab.Save(chain.Selector, consumerResp.Address.String(), consumerResp.Tv)
+	if err != nil {
+		return fmt.Errorf("failed to save FeedsConsumer: %w", err)
+	}
+	lggr.Infof("Deployed %s chain selector %d addr %s", consumerResp.Tv.String(), chain.Selector, consumerResp.Address.String())
```

Comment:
This log line is pretty deep in the call stack. Typically things are cleaner when we return errors from plain `funcs` and log only from method `funcs`, which have a contextual logger attached and which we expect to have these sorts of side effects. How would things look if we logged this line from the caller, and dropped the `lggr logger.Logger` argument?